REVIEW.md

```diff
@@ -0,0 +1,219 @@
+If appropriate, help the contributor understand why tests failed.
+- [ ] Does any added/changed content render correctly, without user-visible issues?
+- [ ] Is the content free from typos and unambiguous grammar errors (review the [Style Guide](https://github.com/spyder-ide/spyder-docs/blob/master/STYLEGUIDE.md) for more information)?
+
+Review that addresses these questions on a PR will be prefixed with `[required]` so that the PR author(s) are clear that they are blockers.
+PR authors are responsible for addressing these reviews if prompted by the reviewer in order for that PR to be eligible for merging.
+
+### Optional review
+
+The following are themes that may be helpful to cover on a documentation-focused PR, but are optional.
+This may include feedback about non-blocking, substantive, user-visible content improvements to the text and images.
+As with all optional review, please primarily consider how a PR would impact documentation stability, clarity, and quality over aiming for subjective improvements.
+This section should be used in limited amounts to avoid overtaxing contributors, but can be used judiciously with authors who’ve successfully contributed previous pull requests to documentation repositories.
+Otherwise, they can be handled in a followup by the reviewer, opened as an issue, or deferred entirely.
+
+Examples include:
+
+- [ ] Could the content or quality of GIFs and screenshots be improved?
+- [ ] Would this topic benefit from more detailed descriptions of specific issues or solutions?
+- [ ] Would this topic benefit from additional elaboration on directly related items within the PR’s scope?
+- [ ] Could new content be restructured to read more clearly or logically for users?
+
+Review that addresses these topics on a PR will be prefixed with `[optional]` so that the PR author(s) are clear that this feedback is not a blocker.
+Addressing this feedback is considered above-and-beyond, and may be more relevant on some PRs than others.
+PR authors may address these reviews, but are not required to do so in order for the PR to be eligible for merging.
+
+### Followup-only items
+
+On all documentation PRs that modify more than a few lines, changes that are less critical/unambiguous, optimization-focused, or those that are not user-visible, should not be suggested on contributor PRs (even with an `[Optional]` tag) unless the author explicitly requests this level of review.
+
+Instead, these changes should be made in a followup pull request by a maintainer or other reviewer.
+The specifics are covered in detail in the [Style Guide](https://github.com/spyder-ide/spyder-docs/blob/master/STYLEGUIDE.md), but generally include:
+
+- [ ] Copyediting (diction, phrasing, verbosity, repetition etc)
+- [ ] Formatting (spacing, line breaks, etc)
+- [ ] reST/Markdown style (roles, directives, markup, etc)
+- [ ] Style guide issues (consistency, precise label text, other nits)
+
+### Rebasing and merging
```

Comment:
@dalthviz and others, what is your take on promoting this to be a top-level section, as opposed to applying to some PRs and not others? Notably, it doesn't require that maintainers always squash PRs (or take other actions), it still leaves it to maintainer judgement, but clarifies the expectations and the tools maintainers have at their discretionary disposal to take care of such cases (as we discussed on spyder-ide/qtpy#277 )

Reply:
Just in case, makes sense to me to have as contents/outline something like this:

* Ettiquete Guideline

   * Communication principles (current _Communication Guidelines_)
   * Expected reviewer behavior

* Reviewing process

   * For code-focused review
      * Required review
      * Optional review

   * For documentation-focused review
      * Required review
      * Optional review
      * Followup-only items
 
* Rebasing and merging process

* References

So basically the following flow: understanding how things should be done while reviewing (PR context, communication, etc), understanding what things should be reviewed and finally how to incorporate changes to the project after doing a review. 

Also, the _Feedback_ section/line could go in the description

However, keeping the current contents outline it could be nice to add a section under _For code-focused review_ specifing the rebasing/merging process in that case, so something like:

* Feedback

* Expected reviewer behavior

* Reviewing process

   * For code-focused review
      * Required review
      * Optional review
      * **Rebasing and merging process**

   * For documentation-focused review
      * Required review
      * Optional review
      * Followup-only items
      * Rebasing and merging process

* Communcation Guidelines

* References

Reply:
I love @dalthviz 's idea, IMO that structure makes more sense and is less duplicative, simpler, clearer, and easier to navigate than the current.

Reply:
I've restructured the document to match what I think was described here. Thanks for taking a look at this from a high level, @dalthviz!

Reply:
Thanks @isabela-pf ! Other than a few minor suggested tweaks, looks good to me.